packages/foreman/nodejs-node-gyp/nodejs-node-gyp.spec

```diff
@@ -1,312 +1,313 @@
+Source91: https://registry.npmjs.org/semver/-/semver-5.7.1.tgz
+Source92: https://registry.npmjs.org/sigmund/-/sigmund-1.0.1.tgz
+Source93: https://registry.npmjs.org/sshpk/-/sshpk-1.16.1.tgz
+Source94: https://registry.npmjs.org/string_decoder/-/string_decoder-1.1.1.tgz
+Source95: https://registry.npmjs.org/tar/-/tar-2.2.2.tgz
+Source96: https://registry.npmjs.org/tough-cookie/-/tough-cookie-2.4.3.tgz
+Source97: https://registry.npmjs.org/tunnel-agent/-/tunnel-agent-0.6.0.tgz
+Source98: https://registry.npmjs.org/tweetnacl/-/tweetnacl-0.14.5.tgz
+Source99: https://registry.npmjs.org/type/-/type-1.2.0.tgz
+Source100: https://registry.npmjs.org/uri-js/-/uri-js-4.2.2.tgz
+Source101: https://registry.npmjs.org/util-deprecate/-/util-deprecate-1.0.2.tgz
+Source102: https://registry.npmjs.org/uuid/-/uuid-3.3.3.tgz
+Source103: https://registry.npmjs.org/verror/-/verror-1.10.0.tgz
+Source104: https://registry.npmjs.org/which/-/which-1.3.1.tgz
+Source105: https://registry.npmjs.org/wrappy/-/wrappy-1.0.2.tgz
+Source106: nodejs-node-gyp-%{version}-registry.npmjs.org.tgz
+Source107: addon-rpm.gypi
+
+Patch1: node-gyp-node-dirs.patch
+
+Requires:   gyp
+Requires:   %{?scl_prefix_nodejs}nodejs-devel
+Requires:   libuv-devel
+Requires:   http-parser-devel
+Requires:   gcc-c++
+Requires:   python-devel
+Requires:   make
+
+BuildRequires: %{?scl_prefix_nodejs}nodejs-devel
+%if 0%{!?scl:1}
 BuildRequires: nodejs-packaging
-BuildRequires: npm
+%endif
 BuildArch: noarch
-%if 0%{?fedora} >= 19
 ExclusiveArch: %{nodejs_arches} noarch
-%else
-ExclusiveArch: %{ix86} x86_64 %{arm} noarch
-%endif
-Provides: npm(%{npm_name}) = %{version}
-Provides: bundled(gyp)
-Provides: bundled(npm(fstream)) = 1.0.10
-Provides: bundled(npm(minimatch)) = 1.0.0
-Provides: bundled(npm(nopt)) = 3.0.6
-Provides: bundled(npm(glob)) = 4.5.3
-Provides: bundled(npm(path-array)) = 1.0.1
-Provides: bundled(npm(semver)) = 5.3.0
-Provides: bundled(npm(rimraf)) = 2.6.1
-Provides: bundled(npm(tar)) = 2.2.1
-Provides: bundled(npm(osenv)) = 0.1.4
-Provides: bundled(npm(which)) = 1.2.12
-Provides: bundled(npm(inherits)) = 2.0.3
-Provides: bundled(npm(mkdirp)) = 0.5.1
-Provides: bundled(npm(npmlog)) = 2.0.4
-Provides: bundled(npm(abbrev)) = 1.1.0
-Provides: bundled(npm(lru-cache)) = 2.7.3
-Provides: bundled(npm(inflight)) = 1.0.6
-Provides: bundled(npm(sigmund)) = 1.0.1
-Provides: bundled(npm(once)) = 1.4.0
-Provides: bundled(npm(request)) = 2.79.0
-Provides: bundled(npm(minimatch)) = 2.0.10
-Provides: bundled(npm(block-stream)) = 0.0.9
-Provides: bundled(npm(array-index)) = 1.0.0
-Provides: bundled(npm(os-homedir)) = 1.0.2
-Provides: bundled(npm(isexe)) = 1.1.2
-Provides: bundled(npm(os-tmpdir)) = 1.0.2
-Provides: bundled(npm(glob)) = 7.1.1
-Provides: bundled(npm(minimist)) = 0.0.8
+
+Provides: %{?scl_prefix}npm(%{npm_name}) = %{version}
+Provides: bundled(npm(abbrev)) = 1.1.1
+Provides: bundled(npm(ajv)) = 6.10.2
 Provides: bundled(npm(ansi)) = 0.3.1
-Provides: bundled(npm(are-we-there-yet)) = 1.1.2
```

Comment:
it confused me that we didn't touch the gyp version, and yet the versions of the bundled modules changed, but in retrospect this is correct, as we pulled new versions that still satisfy the deps.